pytorch_lightning/trainer/connectors/logger_connector/logger_connector_new.py

```diff
@@ -277,6 +278,14 @@ def reset(self, metrics: Optional[bool] = None) -> None:
         self._batch_idx = None
         self._split_idx = None
         self._current_fx = None
+    
+    @property
+    def spawn_callback_metrics(self) -> Dict[MetricSource, Dict[str, _METRIC]]:
```

Comment:
I think this should be more general than just callback_metrics.

``spawn_extra_parameters``.

Reply:
done